file-formats/aifd/examples/z_aff_example_aifd.aifd.json

```diff
@@ -0,0 +1,22 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Example AIFD for ABAP File Format",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "cloudDevelopment"
+  },
+  "aifInterfaces":[
+    {
+      "objectName":"/AIFTCUST_WS_I00001",
+      "namespace":"/AIFT",
+      "interfaceName":"CUST_WS_I",
+      "interfaceVersion":"00001"
```

Comment:
Isn't the data redundant? So wouldn't it be sufficient to either have only the `objectName` or the fields `namespace`, `interfaceName`, `interfaceVersion`?

Reply:
No, the data is not necessarily redundant. It may be that the interface object name is derived from the interface keys namespace, interface name and interface version, but this is not mandatory and can be different if necessary.
And in exactly such a case, when the interface object name is different, an additional display of the interface keys makes sense.

Reply:
What do you mean with use different names?

Reply:
OK, I see. 👍 

> What do you mean with use different names?

Maybe, we suggest different field names. But we have to double-check this aspect first.